Source/Core/VideoCommon/HiresTextures.cpp

```diff
@@ -184,34 +318,56 @@ void HiresTexture::Prefetch()
       }
       if (iter != s_textureCache.end())
       {
-        for (const Level& l : iter->second->m_levels)
+        for (const Level& tex : iter->second->m_levels)
```

Comment:
For what will be one more word that is so similar to a global one, to make things even more confusing? I wasn't able to figure out, evne with help from IRC, what that actually does in technical detail so I has no proper idea for that name. On the other hand we know that this counts all the memory of all textures that are prefetched, that could give some ideas how to name this, the term level simply doesn't mean anything for this local case, or if it does, then I'm open to hearing an explanation.